scripts/config/templates/nginx.conf.erb

```diff
@@ -107,20 +112,25 @@ http {
     proxy_pass https://s3.amazonaws.com/neighbor-build-${environment}/frontend$uri;
   }
 
+  location ~ \.xml$|\.txt$ {
+    # bypass rendertron
+    try_files $uri =404;
+  }
+
   <% routes.each do |route, path| %>
   location ~ ^<%= route %>$ {
     set $route <%= route %>;
     mruby_set $path /app/bin/config/lib/ngx_mruby/routes_path.rb cache;
     mruby_set $fallback /app/bin/config/lib/ngx_mruby/routes_fallback.rb cache;
 
-    location ~ \.xml$ {
-      # bypass rendertron
-      try_files $uri =404;
-    }
-
-    location ~ \.txt$ {
-      # bypass rendertron
-      try_files $uri =404;
+    # Only proxy to prerendered page if:
+    # 1. There is not a neighbor rendertron header
+    # 2. The $request_uri has some kind of :city--:state- match
+    # 3. The $request_uri is on the correct arm of the A/B test
+    set $static_render_condition "${http_x_neighbor_rendertron}---${request_uri}${is_static_rendered}";
+    if ($static_render_condition ~ ^---.*[A-Z][a-z].*--[A-Z][a-z].*-.*true$) {
```

Comment:
Write regex DLP path match that never includes listing detail urls (e.g. https://nbr.pizza:5001/rentals/Asheboro--North-Carolina-self-storage-space-480)